plugins/woocommerce/client/blocks/packages/components/radio-control/option-layout.tsx

```diff
@@ -31,7 +37,12 @@ const OptionLayout = ( {
 				) }
 			</div>
 			{ ( description || secondaryDescription ) && (
-				<div className="wc-block-components-radio-control__description-group">
+				<div
+					className={ clsx(
```

Comment:
Nit (not blocking): Seems a really simple reason to introduce another dependency when you could just do:

`wc-block-components-radio-control__description-group wc-block-components-radio-control__description-group--${ descriptionStackingDirection }`

Reply:
Since we're not applying any styling to `--row` classes, could we change this so we only add `wc-block-components-radio-control__description-group--column` if `descriptionStackingDirection` is `column`? We'd prevent outputting `wc-block-components-radio-control__description-group--row` then.

Though I'm wondering if we would prefer to keep this in case we _do_ want to add styling to `--row` classes in future.

Reply:
I agree in general, but in this case, I would leave it in just for the sake of readability (keeping the lines shorter). `clsx` is not a dependency that we can realistically ever get rid of, so it doesn't feel that bad.

Reply:
I think it's because I added it in bulk as part of the review, sorry for any confusion.

I think the change below is wrong, it should be for `column` right? Since `row` is default and has no styling we only want to do this when it's in column mode.